### ISSUE TITLE
Error when sending large unreliable broadcast

### ISSUE THREAD

**Issue by TsFreddie** [FirstGearGames/FishNet#557]
**General**
Unity version: 2022.3.8f
Fish-Networking version: 3.11.10

**Description**
Got a error when sending a large unreliable broadcast
```
A message of length 1777 requires the reliable channel but was sent on channel Unreliable. Please file this stack trace as a bug report.
UnityEngine.Debug:LogError (object)
FishNet.Managing.Logging.LevelLoggingConfiguration:LogError (string) (at Assets/FishNet/Runtime/Managing/Logging/LevelLoggingConfiguration.cs:128)
FishNet.Managing.NetworkManager:LogError (string) (at Assets/FishNet/Runtime/Managing/NetworkManager.Logging.cs:116)
FishNet.Managing.Transporting.TransportManager:SplitRequired (byte,int,int&,int&) (at Assets/FishNet/Runtime/Managing/Transporting/TransportManager.cs:564)
FishNet.Managing.Transporting.TransportManager:SetSplitValues (byte,System.ArraySegment`1<byte>,bool,int&,int&) (at Assets/FishNet/Runtime/Managing/Transporting/TransportManager.cs:525)
FishNet.Managing.Transporting.TransportManager:SendToClient (byte,System.ArraySegment`1<byte>,FishNet.Connection.NetworkConnection,bool,FishNet.Object.DataOrderType) (at Assets/FishNet/Runtime/Managing/Transporting/TransportManager.cs:407)
FishNet.Managing.Server.ServerManager:Broadcast<RedSnapper.Snapshot.ServerSnapshotBroadcast> (FishNet.Connection.NetworkConnection,RedSnapper.Snapshot.ServerSnapshotBroadcast,bool,FishNet.Transporting.Channel) (at Assets/FishNet/Runtime/Managing/Server/ServerManager.Broadcast.cs:226)
RedSnapper.Snapshot.SnapshotManager:ServerPostTick () (at Assets/RedSnapper/Runtime/Snapshot/SnapshotManager.cs:220)
RedSnapper.Snapshot.SnapshotManager:OnPostTick () (at Assets/RedSnapper/Runtime/Snapshot/SnapshotManager.cs:155)
FishNet.Managing.Timing.TimeManager:IncreaseTick () (at Assets/FishNet/Runtime/Managing/Timing/TimeManager.cs:727)
FishNet.Managing.Timing.TimeManager:<TickUpdate>g__MethodLogic|104_0 () (at Assets/FishNet/Runtime/Managing/Timing/TimeManager.cs:383)
FishNet.Managing.Timing.TimeManager:TickUpdate () (at Assets/FishNet/Runtime/Managing/Timing/TimeManager.cs:373)
FishNet.Transporting.NetworkReaderLoop:Update () (at Assets/FishNet/Runtime/Transporting/NetworkReaderLoop.cs:28)
```

**Expected behavior**
If unreliable broadcast splitting is supported then this is probably a bug. If it isn't then this error message is kinda confusing. Didn't really expected anything, the error message says I should file this as a bug report so I did.

**Comment by FirstGearGames** [comment]
Thanks. That should be handled automatically so yes, filing a report is the right thing to do. Thanks.

**Comment by FirstGearGames** [comment]
Fixed in 3.11.13 and 4.0.6
